docs/knative-eventing-mesh/08-01-configure-kafka-channel.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Configure Kafka Channel
+type: Tutorials
+---
+
+Instead of the default Channel implementation, you can use the Knative-compatible [Kafka Channel](https://github.com/kyma-incubator/knative-kafka). To ensure Kafka works properly, you must:
+
+* Install the `knative-eventing-kafka`  component to deploy the Kafka controller
+* Set up a Kafka cluster using Azure Event Hubs
+* Create a Secret whic the controller uses to communicate with the cluster
+
+**NOTE**: Although Event Hubs and Kafka are nearly identical as concepts, they use different naming convention. To avoid confusion, read [this document](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-for-kafka-ecosystem-overview#kafka-and-event-hub-conceptual-mapping).
+
+## Prerequisites
+
+Before Kyma installation, [enable](/root/kyma/#configuration-custom-component-installation) the `knative-eventing-kafka` custom component.
+
+## Steps
+
+Follow these steps:
+
+1. Use the Azure portal to create a [resource group](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-a-resource-group) where your will place your cluster.
+
+2. Create an [Event Hub namespace](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hubs-namespace) which is an Event Hub representation of the cluster.
+
+  **NOTE**: You can also use other providers, such as [Confluent Cloud](https://www.confluent.io/confluent-cloud) or install Kafka [locally](https://kafka.apache.org/quickstart), but bear in mind that these are experimental options.
+
+3. Export the variables. To retrieve the credentials, go to Azure Portal > **All services** > **Event Hubs** and select your Event Hub. 
+
+  ```bash
+  $ export kafkaBrokers={BROKER_URL}
+  $ export kafkaNamespace={KAFKA_CLUSTER_NAME}
+  $ export kafkaPassword={PASSWORD}
+  $ export kafkaUsername={CONNECTION_STRING}
```

Comment:
`$ConnectionString` is not a variable here, but the real value instead. Confusing, I know. If we apply the overrides with kubectl, we need to escape it (`\$ConnectionString`), otherwise the shell will substitute with the value of the variable. Since it is no variable, we need to escape.

```suggestion
  $ export kafkaUsername=\$ConnectionString
```

I think `kafkaUsername=\$ConnectionString` is unique to Azure, see [comment in knative-kafka code](https://github.com/kyma-incubator/knative-kafka/blob/678d108b3f03b5ba7305095976ebea5114a385c7/components/common/pkg/kafka/admin/admin.go#L37). A few lines below, you can also see that `$ConnectionString` is the actual username.